Bio/SeqIO/QualityIO.py

```diff
@@ -1354,22 +1353,11 @@ def __init__(self, source, alphabet=single_letter_alphabet, title2ids=None):
         EAS54_6_R1_2_1_443_348 ?????????????????????????
 
         Becase QUAL files don't contain the sequence string itself, the seq
-        property is set to an UnknownSeq object.  As no alphabet was given, this
-        has defaulted to a generic single letter alphabet and the character "?"
-        used.
-
-        By specifying a nucleotide alphabet, "N" is used instead:
-
-        >>> from Bio import SeqIO
-        >>> from Bio.Alphabet import generic_dna
-        >>> with open("Quality/example.qual") as handle:
-        ...     for record in SeqIO.parse(handle, "qual", alphabet=generic_dna):
-        ...         print("%s %s" % (record.id, record.seq))
-        EAS54_6_R1_2_1_413_324 NNNNNNNNNNNNNNNNNNNNNNNNN
-        EAS54_6_R1_2_1_540_792 NNNNNNNNNNNNNNNNNNNNNNNNN
-        EAS54_6_R1_2_1_443_348 NNNNNNNNNNNNNNNNNNNNNNNNN
+        property is set to an UnknownSeq object.  Although the sequence is
+        almost certainly DNA we can't be sure, so the character "?" is used
+        rather than "N".
```

Comment:
If it's not DNA, it's RNA; we can be sure it's not protein. So I think it's safe to use N here.

Reply:
We have no idea what the "sequence" is, there is no sequence in a QUAL file, just entries like this:

```
>identifier and optional description
50 50 50 40 20 ...
```

By convention these were used with Roche 454 with a matching DNA FASTA file, but could be provided with any sequence including peptides. In practice though, almost certainly DNA.

Reply:
OK